api_v3/lib/types/serverNode/base/KalturaServerNode.php

```diff
@@ -49,51 +56,52 @@ class KalturaEdgeServer extends KalturaObject implements IFilterable
 	public $systemName;
 	
 	/**
-	 * edgeServer description
 	 * 
 	 * @var string
 	 */
 	public $description;
 	
 	/**
-	 * @var KalturaEdgeServerStatus
-	 * @filter eq,in
-	 */
-	public $status;
-	
-	/**
-	 * edgeServer tags
+	 * serverNode hostName
 	 *
 	 * @var string
 	 * @filter like,mlikeor,mlikeand
 	 */
-	public $tags;
+	public $hostName;
 	
 	/**
-	 * edgeServer host name
-	 * 
-	 * @var string
-	 * @filter like,mlikeor,mlikeand
+	 * @var KalturaServerNodeStatus
+	 * @filter eq,in
 	 */
-	public $hostName;
+	public $status;
```

Comment:
Please make the status read only, is should be set using enable and disable actions.